prettypandas/formatters.py

```diff
@@ -52,22 +58,23 @@ def as_unit(v, unit, precision=2, location='suffix'):
         raise TypeError("Precision must be an integer.")
 
     if location == 'prefix':
-        formatter = partial(format_number, prefix=unit)
+        formatter = partial(format_number, prefix=unit, **kwargs)
     elif location == 'suffix':
-        formatter = partial(format_number, suffix=unit)
+        formatter = partial(format_number, suffix=unit, **kwargs)
     else:
         raise ValueError("location must be either 'prefix' or 'suffix'.")
 
     return formatter(v, "0.{}f".format(precision))
 
+#disable use of babel for percentages so we can control precision
+#as_percent = partial(numbers.format_percent,
+#                     locale=LOCALE_OBJ)
+#"""Format number as percentage."""
```

Comment:
Yeah, it could be a class level variable so someone could do

```
PrettyPandas.set_locale_aware(True)
```
or 
```
PrettyPadnas.set_percent_formatter(as_percent_babel)
```